packages/components/psammead-livel-label/src/index.stories.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import { Headline, Link } from '@bbc/psammead-story-promo';
+import LiveLabel from './index';
+import notes from '../README.md';
+
+const Wrapper = styled.div`
```

Comment:
Why do we need this `Wrapper`? Live label with a headline seems to be displayed in the same way if you remove it

Reply:
I've added it to avoid the whole story view to be clickable.